terra-jupyter-bioconductor/Dockerfile

```diff
@@ -7,4 +7,6 @@ RUN curl -O https://raw.githubusercontent.com/Bioconductor/anvil-docker/master/a
 	&& R -f install.R \
 	&& rm -rf install.R
 
+RUN chown -R $USER:users /opt/conda
```

Comment:
does this need to be in each individual image even tho it's already in base image?